registry/slashFun.js

```diff
@@ -0,0 +1,123 @@
+// @ts-check
+const u = require('./regUtils');
+
+const roll = new u.sub()
+  .setName("roll")
+  .setDescription("Roll Dice")
+  .addIntegerOption(
+    new u.int()
+    .setName("dice")
+    .setDescription("How many dice to roll? (will roll at least 1 anyway) (max of 10000)")
+    .setRequired(false)
+    .setMinValue(1)
+    .setMaxValue(10000)
```

Comment:
```suggestion
    .setDescription("How many dice to roll? (Default: 1)")
    .setRequired(false)
    .setMinValue(1)
    .setMaxValue(500)
```

Reply:
rolling 10000d100000 still takes less than 50 ms in the code... and besides that said reduction would only make it 1/20th of the time... I measured 5 ms for 10000d100000, but it could theoretically vary a bit...

Reply:
👍